### PR TITLE
fix: escape field names in generated code (#45) 

### PR DESCRIPTION
**Pull request by rise0chen**
#45 
Adding Suffix `__` for the variable which named by `proto field`.

Notice: can't add Prefix, beacuse of `__r#type` is invalid.

### CODE REVIEW

File: pbjson-test/protos/duplicate_name.proto [hunk 1]
```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
```
Comment (tustvold) [inline pbjson-test/protos/duplicate_name.proto:1]: I like how thorough this test is :+1:

**Review by tustvold — COMMENTED** [review]
This looks good to me, thank you. Not sure why CI is refusing to run, perhaps it will wake up eventually...

Two minor things:

* Could please sign the [CLA](https://www.influxdata.com/legal/cla/)
* Could you update your commit messages to be [semantic](https://www.conventionalcommits.org/en/v1.0.0/) I've taken the liberty of updating the PR title

**Comment by rise0chen** [comment]
Ok

**Review by tustvold — APPROVED** [review]
Thanks, looks good to me :+1:

Circle CI has run, but the actions reporting appears to be broken - https://app.circleci.com/pipelines/github/influxdata/pbjson?branch=pull%2F47&filter=all

**Pull request merged**